bin/init.bash

```diff
@@ -68,6 +68,13 @@ else
   log_info "Inventory already exists, leaving it as it is"
 fi
 
+# Copy node-labels-and-taints.yaml
+if [[ ! -f "${config[node_labels_and_taints]}" ]]; then
+  PREFIX=${prefix} envsubst >"${config[node_labels_and_taints]}" <"${config_defaults_path}/node-labels-and-taints.yaml"
```

Comment:
Suggestion: Add this as a migration step as well to have it be set for existing clusters

Reply:
I was considering this but I don't know if it is such a good idea because the defaults could be incorrect compared to the existing setups. I wrote about it in the "information to reviewers" section of the PR description:
```
Adding a migration script that introduces the node-labels-and-taints.yaml file might be problematic since we don't really know which node labels and taints should be used.
```

Reply:
New migration doc for v2.28: https://github.com/elastisys/compliantkubernetes-kubespray/pull/442/commits/93d509639e5fd009a7ca66ff696c1486ed40b96b
Instructions for labeling and tainting nodes: https://github.com/elastisys/compliantkubernetes-kubespray/pull/442/commits/67698e7bcc5edd498ef42824e7b3d2dd119c2c34

Reply:
Perfection!